src/plugins/expressions/common/ast/types.ts

```diff
@@ -64,7 +64,7 @@ export type ExpressionAstFunctionDebug = {
   /**
    * Raw error that was thrown by the function, if any.
    */
-  rawError?: any | Error;
+  rawError?: any | Error; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
why do we need any here ?

Reply:
Because the raw error can be anything, and the only correct type here is either `any` or `unknown`. The problem is that the latter is not serializable. I thought it's okay to keep that as an exception since it is valid logically.